.tiltignore

```diff
@@ -4,4 +4,7 @@
 LICENSE
 Makefile
 .github
-.vscode
\ No newline at end of file
+.vscode
+*_test.go
+*/test
+.sh
```

Comment:
NIT can you please add an empty line at the end of the file? 
See the symbol

Reply:
done